tests/heap/main.c

```diff
@@ -30,12 +32,12 @@ main(int argc, char * argv[])
 		exit(1);
 
 	/* Suck in the input. */
-	while ((s = fgetln(stdin, &l)) != NULL) {
```

Comment:
fgetln really is a far cleaner way of doing this.  And it avoids the line length limitation.  Is it really necessary to change this?